internal/clients/team6/judge.go

```diff
@@ -16,6 +16,7 @@ func (j *judge) GetPardonedIslands(currentSanctions map[int][]shared.Sanction) m
 
 	for timeStep, sanctions := range currentSanctions {
 		for who, sanction := range sanctions {
+			pardons[timeStep] = initialiseBoolArray(len(sanctions))
```

Comment:
```suggestion
			pardons[timeStep] = make([]bool, len(sanctions))
```

Default value for bools is `false` :)

Reply:
My god, I'd no idea

Reply:
Thanks, latest commit does this and removes function